### PR TITLE
Add readme file with CLI instructions

### PR DESCRIPTION
**Pull request by nluaces**

### CODE REVIEW

File: cmd/skupper/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_attached_connector_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_certificate_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_connector_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_link_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_listener_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_router_access_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_secured_access_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_site_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/cmd/controller/deploy_cluster_scope.yaml
```
Comment (lynnemorrison) [inline cmd/skupper/README.md:20]: I'm wondering if we should separate installing the CRDs from the actual line that starts up skupper-controller?

File: cmd/skupper/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+# Link sites
+
+Create a AccessGrant in west site and generate a file with the accessToken to redeem in east site:
+
+```
+skupper token issue token ~/token.yaml -n west
+```
+
```
[the remark this answers is not in the document]
Comment (nluaces) [reply cmd/skupper/README.md:72]: done e20a6842fb3f98db96a84d9108566b84d78a066f

File: cmd/skupper/README.md (hunk 1 above)
Comment (nluaces) [reply cmd/skupper/README.md:26]: separated in e20a6842fb3f98db96a84d9108566b84d78a066f

What do you mean by starting the skupper controller? after applying deploy_cluster_scope.yaml, the skupper controller is running, you do not need to do anything else.

**Review by lynnemorrison — APPROVED** [review]
looks great!!

File: cmd/skupper/README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+# Consume backend in west site (create a listener)
+
+```
+skupper listener create backend 8080 --host backend --routing-key backend -n west
+```
+
+# Link sites
+
+Create a AccessGrant in west site and generate a file with the accessToken:
```
Comment (Karen-Schoener) [inline cmd/skupper/README.md:67]: Apologies for the complete nit but: consider updating accessToken -> AccessToken?

File: cmd/skupper/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+# Expose backend in east site (create a connector)
+
+```
+skupper connector create backend 8080 --routing-key backend --selector app=backend -n east
+```
+
+# Consume backend in west site (create a listener)
+
+```
+skupper listener create backend 8080 --host backend --routing-key backend -n west
```
Comment (ajssmith) [inline cmd/skupper/README.md:62]: As this is k8s example, would suggest not having --host as it will pick up the name iiuc

**Review by ajssmith — APPROVED** [review]

**Review by Karen-Schoener — APPROVED** [review]

File: cmd/skupper/README.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+# Link sites
+
+Create a AccessGrant in west site and generate a file with the AccessToken:
+
+```
+skupper token issue token ~/token.yaml -n west
```
Comment (fgiorgetti) [inline cmd/skupper/README.md:70]: Minor change. Maybe rename the token name in the command line above, from just `token` to `west-token`,
`my-token` or something that makes it clear that it is just a name. Wdyt?

**Review by fgiorgetti — APPROVED** [review]
Minor suggestion only.

File: cmd/skupper/README.md (hunk 5 above)
Comment (nluaces) [reply cmd/skupper/README.md]: fixed 9f54f365b517e73e1536302514d4ee88bea1231f

File: cmd/skupper/README.md (hunk 4 above)
Comment (nluaces) [reply cmd/skupper/README.md]: That is correct, but it won't be like that until the pull request associated to the issue #1609 is merged. 

fixed 9f54f365b517e73e1536302514d4ee88bea1231f

**Pull request merged**